sub-packages/bionemo-esm2/src/bionemo/esm2/model/finetune/dataset.py

```diff
@@ -69,17 +69,29 @@ def __init__(
 
     @classmethod
     def from_csv(
-        cls, csv_path: str | os.PathLike, tokenizer: tokenizer.BioNeMoESMTokenizer = tokenizer.get_tokenizer()
+        cls,
+        csv_path: str | os.PathLike,
+        tokenizer: tokenizer.BioNeMoESMTokenizer = tokenizer.get_tokenizer(),
+        ignore_labels: bool = False,
```

Comment:
Why wouldn't we just ignore them in the finetuning datamodule itself? If that's too cumbersome this is fine

Reply:
We would want the labels if we are finetuning. Also, there is a method that transform labels in these datasets and that's going to cause the issue anyway.